### DISCUSSION TITLE
Modemmanager usage

### DISCUSSION THREAD

**Discussion by hazerflac7** [Ysurac/openmptcprouter#2492] [Q&A]
I have a modem USB plug in the serving PC and would like to execute AT commands. Everytime I try to do that it always gives me an error saying I have to be in debug mode which I already set it with mmclii -g debug. Any thoughts?

**Comment by MinhHieuDL** [comment]
I also have the same problem with the modemmanager. I port some packages which contain multiple scripts (use uqmi and AT command) to manage the modem USB. When the router init after reboot , I observe a lot of timeout error from AT command. Checking the log, I got the error log:
```
 **<warn>  [modem0] couldn't query SIM slots: QMI protocol error (94): 'NotSupported'
Wed Nov  9 05:44:14 2022 daemon.warn [8378]: <warn>  [modem0/sim0] couldn't load list of emergency numbers: Failed to parse CRSM query result '+CRSM: 148,8,""'
Wed Nov  9 05:44:14 2022 daemon.warn [8378]: <warn>  [modem0] couldn't load list of own numbers: Couldn't get MSISDN: QMI protocol error (16): 'NotProvisioned'**
```
I try to remove the modemmanager package and see that AT command works well without error timeout
My question is: is this safe to remove the modemmanager package from openMPTCP

**Comment by Network-Traditions** [comment]
With my setup using a [Telit FN980 5G Modem (HWv1.0 )](https://www.telit.com/devices/fn980-and-fn980m-data-cards-support-5g/) in QMI mode hosted in a [USB3.0 M.2 Key B Modem Adapter Enclosure](https://thewirelesshaven.com/shop/mini-pcie-m2-adapters/modem-enclosure/usb3-0-to-ngff-m-2-key-b-4g-5g-modem-adapter-enclosure-with-sim-card-slot-new-style/) using ModemManager, I use the "minicom" package installed within the System-Software tab of OMR.  Then, from the console, I'm able to use "minicom" to send "AT" commands to the modem (/dev/ttyUSB3 in my case).  Check out this [YouTube video](https://youtu.be/DRddwfZ_TBY?t=239) for a demonstration.
